src/pages/contact_us/_offices.tsx

```diff
@@ -577,53 +596,143 @@ export const Offices = () => {
-                                <img src={Dubai} alt="Dubai - Burj Al Arab" />
+                                <img src={Singapore} alt="Singapore - Merlion Park" />
                             </div>
                             <Header as="h4" type="sub-section-title" mt="0.8rem" mb="1.6rem">
-                                {localize('Dubai')}
+                                {localize('Singapore')}
                             </Header>
                         </OfficeHeader>
                     </Flex>
                     <Flex max_width="48.6rem">
                         <BorderBox>
                             <MapContainer>
                                 <LocalizedLink
-                                    to="https://goo.gl/maps/fz9kjz1MLiCekhRy5"
+                                    to="https://goo.gl/maps/4bg5B7dkvUv5Pa11A"
                                     external
                                     target="_blank"
                                     rel="noopener noreferrer"
                                 >
                                     <QueryImage
-                                        alt="Deriv's office location map in Dubai"
-                                        data={data['map_dubai']}
+                                        alt="Deriv's office location map in Singapore"
+                                        data={data['map_singapore']}
                                         height="100%"
                                     />
                                 </LocalizedLink>
                             </MapContainer>
                             <LocalizedLinkText
-                                to="https://goo.gl/maps/fz9kjz1MLiCekhRy5"
+                                to="https://goo.gl/maps/4bg5B7dkvUv5Pa11A"
                                 external
                                 target="_blank"
                                 rel="noopener noreferrer"
                                 mt="0.8rem"
                                 ml="1.6rem"
                             >
-                                <Desktop>
-                                    <p>Office 1902, Jumeirah Business</p>
-                                    <p> Center 1, JLT Cluster G</p>
-                                </Desktop>
+                                <Desktop>80 Robinson Road, #11-03, Singapore 068898</Desktop>
                                 <Mobile>
                                     <StyledMobileAddress>
-                                        Office 1902, Jumeirah Business Center 1, JLT Cluster G
+                                        80 Robinson Road, #11-03, Singapore 068898
                                     </StyledMobileAddress>
                                 </Mobile>
                             </LocalizedLinkText>
                         </BorderBox>
                     </Flex>
+                    <ContinentHeader>{localize('MIDDLE EAST')}</ContinentHeader>
```

Comment:
can we control this via css instead of putting an uppercase text to translate.

Reply:
Could do that yeah 👍